tests/Generator/GeneratorTest.php

```diff
@@ -667,7 +689,7 @@ public function testGetEmptySoapClientStreamContextOptions()
     {
         $instance = self::getBingGeneratorInstance();
 
-        if (PHP_VERSION_ID < 70015) {
+        if (PHP_VERSION_ID < 70107) {
```

Comment:
This breaks [Travis CI](https://travis-ci.org/WsdlToPhp/PackageGenerator/jobs/301931331)?

It might breaks the tests locally but I prefer to pass the Travis CI tests. This is something weird introduced in PHP 7 but I did not take time to look deeper yet, so please keep the old version